### ISSUE TITLE
The "BOOT-EXFAT-NO_NETWORK-MX4SIO" is not working it stops frezes if I start it

### ISSUE THREAD

**Issue by JMaxxL** [israpps/wLaunchELF_ISR#52]
the last release of BOOT-EXFAT-NO_NETWORK-MX4SIO was working great but this one is not working I had to find an old download and go back to that version to get it to work.

**Comment by israpps** [comment]
> the last release of BOOT-EXFAT-NO_NETWORK-MX4SIO was working great but this one is not working I had to find an old download and go back to that version to get it to work. The file date stated it was made in 2005 weird right... anyway I went back to that 2005 version it works.
> 
> The first release was making me re-insert the SD card to get it to be detected. I did how ever get an earlier download last week that was working.

check under Filebrowser > MISC >  Buildinfo
 for the commit hash
 
 The following sample screenshot shows it on yellow
 
 
![image](https://github.com/israpps/wLaunchELF_ISR/assets/57065102/fda7bc6b-31e1-4df7-9bcc-e78af6353b57)

**Comment by JMaxxL** [comment]
I cant get to that because the cursor is not working at ALL

**Comment by JMaxxL** [comment]
its just the default download
![3](https://github.com/israpps/wLaunchELF_ISR/assets/78325531/3078e58d-cbad-40f2-a005-2df6fb7743f3)

but ISRapps

I found out what is causing it... it ONLY happens when my mx4sio is plugged in IF I UNPLUG it no freeze

**Comment by JMaxxL** [comment]
btw I tried zipping it up of course

**Comment by JMaxxL** [comment]
0a70367 will freeze if the MX4SIO is plugged in BUT as long as as I keep it un plugged it acts normal

**Comment by MayanKoyote** [comment]
If you really want to send archives so badly, read to the end of the post you highlighted, and just use .zip archives.

**Comment by JMaxxL** [comment]
did you see the big red highlighted box that is a zipped file was tried multiple times

**Comment by MayanKoyote** [comment]
Why not.
[BOOT-EXFAT-NO_NETWORK-MX4SIO.zip](https://github.com/israpps/wLaunchELF_ISR/files/12371527/BOOT-EXFAT-NO_NETWORK-MX4SIO.zip)

**Comment by israpps** [comment]
test this one...
[WLE_ISR-REVERT_MX4SIOv1.2.zip](https://github.com/israpps/wLaunchELF_ISR/files/12371821/WLE_ISR-REVERT_MX4SIOv1.2.zip)

**Comment by JMaxxL** [comment]
It worked ! thank you

**Comment by JMaxxL** [comment]
yeah these all worked I cant test the D34 firewire one's because I have a 50000n but the other ones work great
[WLE_ISR-REVERT_MX4SIOv1.2.zip]

**Comment by MayanKoyote** [comment]
Any idea why the new MX4SIO driver is causing these freezes when an MX4SIO device is plugged?
